EssentialApp/EssentialAppTests/FeedAcceptanceTests.swift

```diff
@@ -98,4 +118,21 @@ class FeedAcceptanceTests: XCTestCase {
 			["id": "A28F5FE3-27A7-44E9-8DF5-53742D0E4A5A", "image": "http://feed.com/image-2"]
 		]])
 	}
+
+	private func makeCommentsData() -> Data {
+		return try! JSONSerialization.data(withJSONObject: ["items": [
+			[
+				"id": UUID().uuidString,
```

Comment:
Consider hardcoding the `UUID` used when calling `response(for: url)`, that way we can remove an unnecessary operation in this test:
```suggestion
				"id": "2AB2AE66-A4B7-4A16-B374-51BBAC8DB086"
```